group.yml

```diff
@@ -518,3 +518,195 @@ external_scanners:
+        - python-ifaddr
+        - python-vine
+        - python-keystonemiddleware
+        - python-kombu
+        - python-oslo-cache
+        - python-logutils
+        - python-redis
+        - python-html5lib
+        - python-oslo-rootwrap
+        - python-zope-interface
+        - python-werkzeug
+        - python-cachetools
+        - python-tooz
+        - python-pexpect
+        - python-pint
+        - python-neutronclient
+        - python-greenlet
+        - python-keystoneauth1
+        - python-os-traits
+        - python-fixtures
+        - python-alembic
+        - python-tempita
+        - python-SecretStorage
+        - python-sushy-oem-idrac
+        - python-construct
+        - python-keyring
+        - python-stevedore
+        - python-debtcollector
+        - python-webtest
+        - python-ddt
+        - python-ironicclient
+        - python-jsonpath-rw
+        - python-dracclient
+        - python-os-client-config
+        - python-oslo-serialization
+        - python-sushy
+        - python-oslo-metrics
+        - python-pbr
+        - python-simplejson
+        - python-oslo-upgradecheck
+        - python-statsd
+        - python-oslo-policy
+        - python-scciclient
+        - python-oslo-config
+        - python-gevent
+        - python-uhashring
+        - python-openstacksdk
+        - crudini
+        - python-repoze-lru
+        - python-pecan
+        - python-retrying
+        - python-oslo-utils
+        - python-paste
+        - python-soupsieve
+        - python-ironic-lib
+        - python-fasteners
+        - python-colorama
+        - python-markupsafe
+        - python-oslo-i18n
+        - python-decorator
+        - python-oslo-versionedobjects
+        - python-oslo-middleware
+        - python-msgpack
+        - python-pyperclip
+        - pysnmp
+        - python-sqlparse
+        - python-waitress
+        - python-oslo-service
+        - python-keystoneclient
+        - python-kazoo
+        - python-singledispatch
+        - python-dogpile-cache
+        - python-oslo-concurrency
+        - python-proliantutils
+        - python-editor
+        - python-ironic-prometheus-exporter
+        - python-importlib-metadata
+        - python-zope-event
+        - openstack-ironic-python-agent
+        - python-defusedxml
+        - python-munch
+        - python-oslo-reports
+        - python-oslo-log
+        - python-requests-unixsocket
+        - python-requestsexceptions
+        - python-funcsigs
+        - python-oslo-messaging
+        - python-wrapt
+        - python-zake
+        - python-paste-deploy
+        - python-cinderclient
+        - openstack-ironic
+        - python-simplegeneric
+        - python-voluptuous
+        - python-wsme
+        - python-beautifulsoup4
+        - python-yappi
+        - python-jinja2
+        - python-webob
+        - python-cmd2
+        - python-futurist
+        - python-automaton
+        - python-cliff
+        images:
+        # Built by CPAAS
```

Comment:
Nothing that uses CPaaS needs our scan.
I expect that these images are shipped by other teams via errata other than the ones ART creates (i.e. ART will never attach them to errata). If ART doesn't ship the container, then we should not be scanning it. It is up to that product team to have a secure lifecycle (CPaaS, for example, runs scans).

Reply:
It was there in our candidate tags though. According to current policy, we need to scan everything in our tags

Reply:
But if you feel that we can exclude these, I'll give Hardik a heads up and will exclude them from our pipeline